specifications/api/swagger.yaml

```diff
@@ -1887,12 +1887,505 @@ x-responsesErr: &responsesErr
       description: A specific error code for this error, more precise than HTTP ones.
       example: an_error_code
 
+x-errNotFound: &errNotFound
+  <<: *responsesErr
+  title: not_found
+  properties:
+    message:
+      type: string
+      description: A descriptive error message.
+    code:
+    code:
+      type: string
+      enum: ['not_found']
+
+x-errNoSuchWallet: &errNoSuchWallet
+  <<: *responsesErr
+  title: no_such_wallet
+  properties:
+    message:
+      type: string
+      description: |
+        May occur when a given walletId does not match with any known
+        wallets (because it has been deleted, or has never existed).
+    code:
+      type: string
+      enum: ['no_such_wallet']
+
+x-errNoSuchTransaction: &errNoSuchTransaction
+  <<: *responsesErr
+  title: no_such_transaction
+  properties:
+    message:
+      type: string
+      description: May occur when a given transactionId does not match with any known transactions.
+    code:
+      type: string
+      enum: ['no_such_transaction']
+
+x-errTransactionNotPending: &errTransactionNotPending
+  <<: *responsesErr
+  title: transaction_not_pending
+  properties:
+    message:
+      type: string
+      description: May occur when trying to forget a transaction that is not pending.
```

Comment:
Just to note this will change after https://github.com/input-output-hk/cardano-wallet/pull/2262/files#diff-a2398827a22606930a9b4edc67dfc9c99196a94cdb56c56ef0ef5f92512d42e0R2467 merged. Also expired transaction will be "forgettable".